crates/bevy_picking_core/src/lib.rs

```diff
@@ -53,9 +52,19 @@ impl Default for PickingPluginsSettings {
     }
 }
 
-/// Used to mark entities that should be pickable.
-#[derive(Component, Debug, Default, Clone, Reflect)]
-pub struct Pickable;
+/// Used to mark entities that should be pickable. Similar to bevy's `FocusPolicy`, except this type
+/// allows entities to be completely ignored in the picking process, and does not require compiling
+/// bevy_ui.
```

Comment:
I think it's nice to just have the first sentence separated by a line break because rustdoc will truncate everything after the line break for the abbreviated text that appears next to doc items.